docs/setup-guide.mdx

```diff
@@ -2,53 +2,63 @@
 
-<center><img src={require("../static/img/setup-guide/open-with-vscode.png").default} /></center>
+In this guide, we will introduce how to create a roblox-ts game.
 
-3. Run `npm init roblox-ts` in your command line to start the interactive project setup.
+Make sure you have an up-to-date version of Node.js installed and your current working directory is the one where you intend to create a project. Run the following command in your command line:
 
-:::caution PowerShell
-If you're using PowerShell for the first time, you may get an error about "running scripts is disabled on this system". You can fix this by running the following in PowerShell as an administrator:
+```bash npm2yarn
+npm init roblox-ts
+```
 
-`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
+The tool will guide you through some questions to set up your project:
+1. `Project directory` - What is the name of the folder you want to put your project in? This folder shouldn't exist yet, the tool will create it for you!
+2. `Select template` -  What type of roblox-ts project is this? roblox-ts can be used to build plugins and packages too, but in this case we want to make a game, so press `Enter` once.
+3. `Configure Git` - Do you want to use [Git](https://git-scm.com/) for version control? Press `Y` or `N` to make your choise.
+4. `Configure ESLint` - Do you want to use [ESLint](https://eslint.org/) to check your code for correctness?
+5. `Configure Prettier` - Do you want to use [Prettier](https://prettier.io/) to format your code?
+6. `Configure VSCode Project Settings` - Do you want the tool to create a `.vscode` folder with settings optimized for roblox-ts development? If you're using VSCode to write code, you should use this.
 
-[More info here](./articles/powershell)
+:::tip
+To use ESLint and Prettier within VSCode, you can install their VSCode extensions which directly integrate their features into VSCode.
+- [ESLint VSCode extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+- [Prettier VSCode extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
 :::
 
-4. Start roblox-ts in watch mode via `npx rbxtsc -w`. You can exit watch mode later by clicking on the command line and pressing `Ctrl+C` on your keyboard.
+Then, open your project in your code editor of choice, such as VSCode:
 
-5. Next, start up a Rojo server. If you are using the Rojo VSCode extension, simply hit `Ctrl+Shift+P` and select `Rojo: Start Server`.
+```bash title="Terminal"
+code project-name
+```
 
-<center><img src={require("../static/img/setup-guide/rojo-extension.png").default} /></center>
+## Running Your Code
 
-Otherwise, open a new terminal with the same working directory and start Rojo via `rojo serve`. In VSCode a new terminal can be opened by pressing the `+` button.
+roblox-ts has a feature called "watch mode" where it will continuously build your code as you write it. You can start watch mode with:
```

Comment:
Similar to Rojo, this should include a recommendation for the VSC roblox-ts extension.